i18n/zh-Hans/code.json

```diff
@@ -1,59 +1,59 @@
 {
-  "Get one-on-one support": {
+  "Get one-on-one guidance": {
     "message": "获取一对一操作指导"
   },
-  "Please open this site on PC": {
-    "message": "请使用PC端打开此页面"
+  "Please open this page from PC": {
+    "message": "请从PC端打开此页面"
   },
-  "Please be sure to read the following carefully": {
+  "Please read the following carefully": {
     "message": "请务必认真阅读以下内容"
   },
   "Decline": {
     "message": "拒绝"
   },
-  "Agree": {
-    "message": "同意"
+  "Accept": {
+    "message": "接受"
   },
   "Data Acquisition": {
     "message": "数据采集"
   },
-  "Automated collection of protocol and driver's value": {
-    "message": "无论是协议规定的数据，还是驱动返回值，Shifu都能自动采集"
+  "Automatically capture data from various protocols and drivers": {
+    "message": "自动采集各类协议和驱动的数据"
   },
   "Open Source Community": {
     "message": "开源社区"
   },
-  "Open source with limitless cloud carrier choice Versatile deployment on enterprise, edge, private or public cloud service": {
+  "Shifu is open source and no vendor lock-in. Support on-prem, edge, private, hybrid and public cloud deployment": {
     "message": "Shifu的核心代码全部开源，让您避免供应商锁定。您更可以自由地选择让Shifu部署在企业内部，边缘云、私有云、混合云或公有云上"
   },
-  "K8s Native": {
+  "Kubernetes Native": {
     "message": "K8s原生"
   },
-  "Rooted in K8s, hassle-free migration in the future": {
-    "message": "k8s已经一统天下，Shifu基于k8s的架构可以让你免去未来迁移到k8s上的烦恼"
+  "Shifu inherits kubernetes' cloud-native architecture, enabling IoT Devops with platform-level stability and reliability": {
+    "message": "Shifu基于k8s的云原生架构，以平台级的稳定性和可靠性赋能物联网应用的开发运维"
   },
   "Lightweight Hardware Requirement": {
-    "message": "轻量化"
+    "message": "轻量级硬件要求"
   },
   "As light as a Raspberry Pi! (2G RAM)": {
-    "message": "只需一个2G内存的树莓派就可以在生产环境跑起Shifu"
+    "message": "只需一个2G内存的树莓派就可以运行Shifu！"
   },
-  "Cloud-Side Collaboration": {
+  "Cloud-Edge Collaboration": {
     "message": "云边协同"
   },
-  "Compatible with all K8s-based cloud/edge frameworks (OpenYurt, KubeEdge, etc.)": {
+  "Compatible with all Kubernetes-based cloud/edge frameworks (OpenYurt, KubeEdge, etc.)": {
     "message": "Shifu兼容所有基于k8s的云边协同框架（OpenYurt, KubeEdge等）"
   },
   "Infinite Scalability": {
     "message": "无限扩容"
   },
-  "Billions level of device service scalability Big enough to run the city": {
+  "The kubernetes-based architecture allows Shifu to expand infinitely, easily handle billions of devices integration, even for smart city projects": {
     "message": "基于k8s的架构让Shifu可以无限扩容，轻松处理亿级设备接入，连智慧城市项目都不在话下"
   },
-  "Orchestration for IoT Application Development": {
-    "message": "让开发产业场景像开发APP一样简单！"
+  "Kubernetes native reliability for IoT application development as easy as developing web apps": {
```

Comment:
this sentence seems ungrammatical

 to render IoT application development as easy as developing web apps

Reply:
this line is outdated